variant-inc/terraform-aws-sns-subscription-sqs
# Feature/cloud 1484 minor fixes

# Description

Minor fixes

Fixes [#3](https://usxtech.atlassian.net/browse/CLOUD-1484)

## Type of change


## How Has This Been Tested?

```yaml
Pre reqs:
  - Clone/Pull branch
  - Create sns topic using  sns module from other PR https://github.com/variant-inc/terraform-aws-sns-topic/pull/2
  - Create terraform.tfvars.json 
{
    "name": "test-queue-naveen",
    "topic_arn": "arn:aws:sns:us-east-1:786352483360:dpl-test-topic-naveen",
    "kms_key_sns_alias_arn":"arn:aws:kms:us-east-1:786352483360:key/255fb0cb-ac0b-4237-acf8-93ffd62c988e"
}

  - Create provider.tf (provided in readme.md)

Test Case 1 [Creates SQS and SNS subscription]:
- Steps
  - Terraform init
  - Terraform plan/apply
- Result
 - Should create sns topic

Test Case 2 [Destroy SQS]:
- Steps
  - terraform destroy
- Results
  - See Bucket deleted
```



## Checklist
